addons/mail/models/mail_mail.py

```diff
@@ -160,6 +161,7 @@ def process_email_queue(self, ids=None):
             res = self.browse(ids).send(auto_commit=auto_commit)
         except Exception:
             _logger.exception("Failed processing mail queue")
+        self.env['mail.mail']._gc_mail_mail()
```

Comment:
Isn't it better to create a cron_trigger for the autovacuum CRON, that way it's run asynchronously?

Reply:
`process_email_queue` is already executed in a CRON, so I'm not sure

It would be more useful if we want to trigger the CRON after sending an email manually (not by the CRON) but we don't do it

Reply:
Still think this is a bit strange.

Either we create a CRON trigger, either we don't call the method and the CRON will be ran at some point anyway.
Is there a real purpose to calling it manually here?